gaas-ui/src/components/user-guide/user-guide.tsx

```diff
@@ -22,7 +22,7 @@ export default class UserGuide extends React.Component<{}, {}> {
 
```

Comment:
When I try to run locally I get an error from line 10 of this file?

Reply:
I've taken a look at this. Seems like it was being caused by some styling however, the styling doesn't actually do anything so I have removed it.

Reply:
Sounds good, works now just getting a warning because the `makeStyles` import is still there

Reply:
Sorry yeah removed that import now